custom_components/localtuya/pytuya/__init__.py

```diff
@@ -1083,32 +1105,34 @@ def _generate_payload(self, command, data=None, gwId=None, devId=None, uid=None)
 
         if command in payload_dict[self.dev_type]:
             if "command" in payload_dict[self.dev_type][command]:
-                json_data = payload_dict[self.dev_type][command]["command"]
+                json_data = payload_dict[self.dev_type][command]["command"].copy()
```

Comment:
Is there a specific need to copy the data; ie you are doing modification to it?

Reply:
```
2023-03-05 12:19:51.182 ERROR (MainThread) [custom_components.localtuya.common] [bfa...vqp] Initial state update failed, giving up: AttributeError("'int' object has no attribute 'copy'")
```
(Not sure if its specifically this copy() or another spot)

Reply:
It's the copy() on command_override. that doesn't return JSON, but just a value. Will update.